tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -237,6 +241,43 @@ function get_tdx_ovmf_path_from_distro() {
 	esac
 }
 
+function adjust_qemu_cmdline() {
+	shim="${1}"
+	config_path="${2}"
+	qemu_share="${shim}"
+
+	# The paths on the kata-containers tarball side look like:
+	# ${dest_dir}/opt/kata/share/kata-qemu/qemu
+	# ${dest_dir}/opt/kata/share/kata-qemu-snp-experimnental/qemu
+	[[ "${shim}" =~ ^(qemu-snp|qemu-nvidia-snp)$ ]] && qemu_share=${shim}-experimental
+		
+	qemu_binary=$(tomlq '.hypervisor.qemu.path' ${config_path} | tr -d \")
+	qemu_binary_script="${qemu_binary}-installation-prefix"
+	qemu_binary_script_host_path="/host/${qemu_binary_script}"
+
+	if [[ ! -f ${qemu_binary_script_host_path} ]]; then
+		# From the QEMU man page:
+		# ```
+		# -L  path
+		# 	Set the directory for the BIOS, VGA BIOS and keymaps.
+		# 	To list all the data directories, use -L help.
+		# ```
+		#
+		# The reason we have to do this here, is because otherwise QEMU
+		# will only look for those files in specific paths, which are
+		# tied to the location of the PREFIX used during build time
+		# (/opt/kata, in our case).
+		cat <<EOF >${qemu_binary_script_host_path}
+#!/usr/bin/env bash
+
+${qemu_binary} "\$@" -L ${dest_dir}/share/kata-${shim}/qemu/
```

Comment:
Does this need to be `${dest_dir}/share/kata-${qemu_share}/qemu/`, or have I misunderstood the logic?